src/management-system-v2/lib/process-export/export-preparation.ts

```diff
@@ -32,10 +37,9 @@ export type ProcessExportData = {
   definitionName: string;
   versions: {
     [version: string]: {
+      name?: string;
       bpmn: string;
-      artefactsToExport: {
-        userTaskFiles: string[];
-      };
+      subprocesses: { id: string; name: string }[];
```

Comment:
Shouldn't name of subprocesses be optional? We can create subprocesses without a name and I think the code in https://github.com/PROCEED-Labs/proceed/pull/122/files#diff-8b837a5efac5d59616e114e2597fc69e634cf026fa79bbac76a9be2c80c60449R109 implies this too.